APBaby/ResourceHelper/APCustomPlayerResourceHelper.h

```diff
@@ -0,0 +1,76 @@
+//
+//  APCustomPlayerResourceHelper.h
+//  Pods
+//
+//  Created by Miri on 09/11/2016.
+//
+//
+
+@import UIKit;
```

Comment:
this is a .h file and not a resource, hence it should be located with all your code files.